composer.json

```diff
@@ -7,17 +7,18 @@
     "license": "MIT",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.4",
-        "doctrine/orm": "^2.10",
-        "symfony/form": "^5.3|^6.0|^7.0",
-        "symfony/framework-bundle": "^5.3|^6.0|^7.0"
+        "php": "^8.1",
+        "doctrine/orm": "^3.0",
```

Comment:
@alioguz Do you try to allow  orm 2 and dbal 2 ?

Reply:
Yes, I tried this config : 

`
"php": "^8.1",
"doctrine/orm": "^2.0",
"doctrine/dbal": "^2.0",
`

But an error occurred when I launch `composer test` : 

`PHP Fatal error:  Declaration of Doctrine\Bundle\DoctrineBundle\DependencyInjection\DoctrineExtension::getMappingResourceConfigDirectory(): string must be compatible with Symfony\Bridge\Doctrine\DependencyInjection\AbstractDoctrineExtension::getMappingResourceConfigDirectory(?string $bundleDir = null): string in /home/ali/dev/form-filter-bundle/vendor/doctrine/doctrine-bundle/DependencyInjection/DoctrineExtension.php on line 868`